### ISSUE TITLE
Keyboard-only navigation does not work in Contact list, Settings dialog, shortcut cheatsheet dialog, etc.

### ISSUE THREAD

**Issue by dumblob** [deltachat/deltachat-desktop#2784]
- Operating System (Linux/Mac/Windows/iOS/Android): Linux
- Delta Chat Version: 1.29.1 (compiled from git 14743055 )

- Expected behavior: keyboard-only navigation works in all dialogs, windows, etc.

- Actual behavior: E.g. in the Contact list, Settings dialog, shortcut cheatsheet dialog, etc. scrolling works only with mouse wheel but not with page up/down nor with arrow up/down. E.g. pressing Tab does not move focus to another user-interactable item/widget in the app so it is impossible to control Delta Chat Desktop without any pointing device and second it is impossible to learn a specific quick workflow using only keyboard.

- Steps to reproduce the problem: Open DC Desktop and press Tab any number of times (at least once) and then try page up/down or arrow up/down but nothing happens. Close DC Desktop and start it again, then click on hamburger, then click on `Settings` and finally press PgUp/PgDown/ArrowUp/ArrowDown/... and nothing happens.

- Screenshots: N/A
- Logs: N/A

**Comment by WofWca** [comment]
> scrolling works only with mouse wheel but not with page up/down nor with arrow up/down

Fixed in https://github.com/deltachat/deltachat-desktop/pull/4269

> Settings dialog

https://github.com/deltachat/deltachat-desktop/pull/4319

Keyboard navigation has been significantly improved in 1.48. The biggest thing remaining now is https://github.com/deltachat/deltachat-desktop/pull/4294, and I guess we have https://github.com/deltachat/deltachat-desktop/issues/2141 to track that.
So, I suggest to close this issue as resolved.
